api/src/main/java/com/staxpayments/api/models/Invoice.kt

```diff
@@ -0,0 +1,42 @@
+package com.staxpayments.api.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+
+@Serializable
+data class Invoice(
+    val id: String,
+    val total: Double?,
+    val meta: JsonElement?,
+    val status: String?,
+    val url: String?,
+    val reminder: JsonElement?,
+    val schedule: JsonElement?,
+    val customer: Customer?,
+    val user: User?,
+    val files: JsonElement?,
+    @SerialName("child_transactions") val childTransactions: List<JsonElement>?,
```

Comment:
change the type to `JsonElement?` instead of `List<JsonElement>?`

Reply:
That was deliberate, I was actually waiting on this PR to be merged #83.
The type there is a Transaction type. I have updated it.